### PR TITLE
Hotfix for `CLIPVisionModelWithProjection`

### PR DESCRIPTION
**Pull request by pcuenca**
This is a quick solution. A more principled way to do it would be to create dummy objects based on version strings.

### CODE REVIEW

File: src/diffusers/utils/import_utils.py [hunk 1]
```diff
@@ -347,3 +347,15 @@ def is_torch_version(operation: str, version: str):
...
+def is_transformers_version(operation: str, version: str):
+    """
+    Args:
+    Compares the current Transformers version to a given reference with an operation.
+        operation (`str`):
+            A string representation of an operator, such as `">"` or `"<="`
+        version (`str`):
+            A string version of PyTorch
+    """
+    return compare_versions(parse(_transformers_version), operation, version)
```
Comment (anton-l) [inline src/diffusers/utils/import_utils.py:361]:
```suggestion
    if not _transformers_available:
        return False
    return compare_versions(parse(_transformers_version), operation, version)
```

**Comment by keturn** [comment]
Should the dependency version get updated?

https://github.com/huggingface/diffusers/blob/f07a16e09bb5b1cf4fa2306bfa4ea791f24fa968/setup.py#L107

**Comment by keturn** [comment]
oh, I see. transformers 4.25 isn't out yet. Well that's awkward.

**Comment by pcuenca** [comment]
> Should the dependency version get updated?
> 
> https://github.com/huggingface/diffusers/blob/f07a16e09bb5b1cf4fa2306bfa4ea791f24fa968/setup.py#L107

Good point. Not sure. We could still request the same version but just disable the two offending pipelines, no reason for others to update.

**Comment by patrickvonplaten** [comment]
Issue is fixed see: https://github.com/huggingface/diffusers/issues/1377

**Comment by anton-l** [comment]
Fixed in `diffusers==0.8.1`: https://github.com/huggingface/diffusers/releases/tag/v0.8.1

**Pull request closed without merging**